package.json

```diff
@@ -3,11 +3,16 @@
   "version": "2.0.0-prealpha.0",
   "description": "Teacher Moments: Practice spaces for teacher preparation programs",
   "main": "src/index.js",
+  "private": true,
+  "workspaces": [
+    "src/client",
+    "src/server"
+  ],
   "scripts": {
     "start": "webpack-dev-server --mode development",
     "build": "webpack --mode development",
     "lint": "eslint src/**/*.js",
-    "dev": ". ./config/pg.dev; nodemon index.js",
+    "dev": ". ./config/pg.dev; nodemon ./src/server/index.js",
```

Comment:
@ktbee I pushed this commit to your branch. Pull this branch again to get the server change and try `yarn dev` to verify the server works

Reply:
Thanks, good catch @evmiguel!